tests/trustyai/utils.py

```diff
@@ -0,0 +1,84 @@
+from typing import List
+
+from ocp_resources.maria_db import MariaDB
+from ocp_resources.mariadb_operator import MariadbOperator
+from ocp_resources.pod import Pod
+from ocp_utilities.operators import TIMEOUT_5MIN
+from timeout_sampler import TimeoutSampler, TimeoutExpiredError
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+def wait_for_mariadb_operator_pods(mariadb_operator: MariadbOperator, timeout: int = 300) -> None:
```

Comment:
i would expect them to be managed by deployments. is this correct? is so, you can get the deployment and wait for replicas

Reply:
Changed this function to wait for deployment replicas